### DISCUSSION TITLE
react-native ??

### DISCUSSION THREAD

**Discussion by smolleyes** [family/connectkit#102] [Q&A] [answered]
hi

can we use this lib with react-native ? 

thanks

**Comment by lochie** [comment, marked as answer]
Currently this library does not support react-native out of the box, but we've seen [an example](https://twitter.com/pugson/status/1612487224058150913) of it being possible.
We haven't been given the source code to the example to engineer this solution into the module itself, so for now this is just a standard react module until further notice.

**Comment by smolleyes** [comment]
Hi

thanks a lot for your reply, getting an "easy" way to connect to wallets on react-native is a real pain, wish connectkit will solve it ! GREAT WORK :)

**Comment by smolleyes** [comment]
hi!

any progress on this :) ?

**Comment by clayrisser** [comment]
Would love to see the code for this!

**Comment by clayrisser** [comment]
Any updates on this?
